coding-concepts.js

```diff
@@ -10,37 +10,46 @@
 var cohort = "Alpha 2022"
 // console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
-
-
+// a) Your answer: I think that it will split the stirng into an array. Either by letter or the whole word.
+// b) Verify and explain: I was right because normally you would use split to turn a string into an array. An array into a string with .join. One thing that you would have to pay attention to is if the quotation marks have a space in between. Those will give you two different answer.
+// console.log(cohort.split("")) WITHOUT SPACE
+// OutPut--->
+//  [
+//   'A', 'l', 'p', 'h',
+//   'a', ' ', '2', '0',
+//   '2', '2'
+// ]
+
+// console.log(cohort.split(" ")) WITH SPACE
+// Output --->
+// [ 'Alpha', '2022' ]
 // --------------------2) What will this log?
 
 const greeter = (name) => {
   `Hello, ${name}!`
 }
 // console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: So this is something that I would like to point... I think this is a dynamic code because you can add any name and you will still expect a string that says hello "name". But i would expect the answer to say "hello LEARN Student"
+// b) Verify and explain: So my predictions were wrong but I would say half wrong. A function will always need a return. If you look at the example given there is no return so when i went to invoke the code I recieved an undefined. Once I added the return I knew right away why I was wrong.
```

Comment:
Totally!  It is so easy to miss some of these small (but big) things like a return.  It happens all the time in coding, but recognizing the error helps to know where to trace back to.